client/bindings/pillarEditor.js

```diff
@@ -17,12 +26,12 @@ define(
             },
             update: function(element, valueAccessor, allBindings, viewModel, bindingContext) {
                 var editing = ko.unwrap(valueAccessor());
-                var pillarModel = bindingContext.$parents[2];
-
+                var pillarModel = getPillarModel(bindingContext);
                 if (editing) {
                     pillarModel.tableEditing = true;
                 }
-                var index = pillarModel.checkedNodes.indexOf(bindingContext.$parent);
+                var hasProjectIndex = pillarModel.checkedNodes.indexOf(bindingContext.$parent);
```

Comment:
the variable name `hasProjectIndex` seems to indicate that it is a boolean, but it is actually an integer, no?